client/pages/confirmation-page.jsx

```diff
@@ -77,12 +79,34 @@ export default class ConfirmationPage extends React.Component {
     fetch('/confirmationPage', req)
       .then(res => res.json())
       .then(order => {
-        const { checkOut } = this.context;
         this.setState({ order });
+      })
+      .catch();
+    this.setState({ salesTax: 0 });
```

Comment:
We should a) print the error at least for debugging purposes and b) wait until our fetch completes to set our state